src/app/components/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { IoMdMail } from "react-icons/io";
+import { FaLinkedin } from "react-icons/fa";
+
+const year = new Date().getFullYear();
+
+const Footer: React.FC = () => {
+  return (
+    <footer style={{
+        width: '100%',
+        backgroundColor: '#402B6D', 
+        color:'#C0F862',
+        padding: '3rem ',
+        fontFamily: '"Roboto Mono", monospace'
+        }}
+    >
+        <div style={{display: 'flex', justifyContent: 'space-between'}}>
+            <button style={{
+                height:'min-content',
+                backgroundColor:'#C0F862',
+                color:'#402B6D',
```

Comment:
Can you use the tailwind classes instead or move these styles into their own file? They shouldn't be hardcoded here. Let me know if you need help with that!